.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report incorrect behavior in the pandas library
+title: "BUG: "
+labels: [Bug, Needs Triage]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the
+            [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the master branch of pandas.
+  - type: textarea
+    id: example
+    attributes:
+      label: Reproducible Example
+      description: >
+        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) on how to
+        provide a copy-pastable example that can be run on a Python REPL.
```

Comment:
`REPL` might be a bit of a confusing term for newer users - also IMO it's more convenient when bug reports are designed to be runnable in one chunk instead of line-by-line (but others may disagree :)

Reply:
Also if you could add `minimal` (like you wrote in the performance version) - hopefully increases likelihood the guide is read and large code snippets aren't just copy-pasted